docs/io/votable/table_element.rst

```diff
@@ -0,0 +1,245 @@
+.. doctest-skip-all
```

Comment:
Ideally we should not skip these. But it's beyond scope of this PR, and certainly is beyond scope this close to feature freeze